internal/server/config/config.go

```diff
@@ -232,7 +232,8 @@ type Config struct {
 	Metrics         MetricConfig
 	CheckQueryCache CheckQueryCache
 
-	RequestDurationDatastoreQueryCountBuckets []string
+	RequestDurationDatastoreQueryCountBuckets    []string
+	RequestDurationDatastoreDispatchCountBuckets []string
```

Comment:
datastore dispatch seems a bit misleading - as dispatch is focused on resolving subgraphs, not datastore lookup.